index.html

```diff
@@ -0,0 +1,791 @@
+                <div class="arrow-button">
+                    <span><i class="fas fa-arrow-right"></i> </span>
+                </div>
+            </div>
+
+            <div class="smash-memb">
+                <div class="memb-text">
+                    <p>
+                        <span> Lorem ipsum dolor sit amet consectetur adipisicing elit. Totam impedit quis fugit iste
+                            provident illo vero sequi doloremque possimus ipsa in ullam eius vel saepe sunt.</span>
+                    </p>
+                    <ul>
+                        <li>
+                            <span> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nihil alias soluta
+                                quia.</span>
+                        </li>
+                        <li>
+                            <span> lorem lorem lorem lorem lorem lorem lorem </span>
+                        </li>
+                        <li>
+                            <span>
+                                lorem lorem lorem lorem lorem
+                            </span>
+                        </li>
+                        <li>
+                            <span>
+                                lorem lorem lorem lorem lorem
+                            </span>
+                        </li>
+                    </ul>
+                    <button type="button">
+                        <span> lorem lorem lorem lorem lorem </span>
+                    </button>
+                </div>
+                <div class="pic-memb-1">
+                    <img class="imgmain" src="img/grey-user.png" alt="">
+                </div>
+            </div>
+
+        </main>
+        <section>
+            <div class="latest-posts">
+                <div class="flex-box-1 classhover">
+
+                    <span>
+                        Lorem Ipsum
+                    </span>
+                    <h2>
+                        <a href="#">
+                            <span> Lorem Ipsum Lorem Ipsum Lorem Ipsum Lorem Ipsum </span>
+                        </a>
+                    </h2>
+                    <article>
+                        <h4>
+                            <span> by </span> <a href="">
+                                <span> Lorem Ipsum </span>
+                            </a>
+                        </h4>
+
+                        <span> Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatum magnam esse, est odit
+                            ullam nemo laboriosam aliquid amet unde, voluptatem tempore! Cum, ut! Minima animi itaque
+                            architecto, corrupti at esse. </span>
+                    </article>
+                    <div class="arrow-button">
+                        <span><i class="fas fa-arrow-right"></i> </span>
+                    </div>
+
+                </div>
+                <div class="flex-box-2 classhover">
+                    <h2>
+                        <span>
```

Comment:
## Article tag

It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

- Change all `<div class="flex-box- classhover">` tags to `<article>` tag in HTML file

Reply:
It's done.